provision/provision.sh

```diff
@@ -559,19 +559,18 @@ services_restart() {
 
 wp_cli() {
   # WP-CLI Install
-  if [[ ! -d "/srv/www/wp-cli" ]]; then
+  local exists_wpcli
+
+  exists_wpcli="$(which wp)"
+  if [[ "/usr/local/bin/wp" != "${exists_wpcli}" ]]; then
     echo -e "\nDownloading wp-cli, see http://wp-cli.org"
-    git clone "https://github.com/wp-cli/wp-cli.git" "/srv/www/wp-cli"
-    cd /srv/www/wp-cli
-    composer install
+    curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli-nightly.phar
+    chmod +x wp-cli.phar
+    sudo mv wp-cli.phar /usr/local/bin/wp
   else
     echo -e "\nUpdating wp-cli..."
-    cd /srv/www/wp-cli
-    git pull --rebase origin master
-    composer install
+    wp cli update --nightly --yes
```

Comment:
If we add something like:
```
if [[ -L "/usr/local/bin/wp" ]]; then
  echo "Removing old wp-cli"
  rm -rf /usr/local/bin/wp /srv/www/wp-cli
fi
```
before `if [[ "/usr/local/bin/wp" != "${exists_wpcli}" ]]; then` it should migrate people seemlessly.

THE ABOVE SNIPPET IS UNTESTED.